cartoframes/contrib/vector.py

```diff
@@ -313,13 +348,15 @@ def __init__(self, dataframe, color=None, size=None, time=None,
             legend=legend
         )
 
-
 @utils.temp_ignore_warnings
 def vmap(layers,
          context,
+         carto_vl_path=_DEFAULT_CARTO_VL_PATH,
```

Comment:
We should document these two arguments btw

Reply:
We can also hide these arguments but still make them accessible by making them `**kwargs`. In the code we can check to see if they're set and override them if not. This way they won't show up in the public documentation but we can still document them in the local dev documentation.

Something like this:
```python
def vmap(layers, context, size=(1024, 632), basemap=BaseMaps.voyager, bounds=None, **kwargs):
    _carto_vl_path = kwargs.get('carto_vl_path', _DEFAULT_CARTO_VL_PATH)
```

Reply:
I like this idea, I don't think there's going to be many use cases for using custom VL / Airship versions

Reply:
I totally agree, actually _javitonino_ suggested the same the other day. I'll change that! 👍